.claude/docs/title-catalog-simple.md

```
@@ -0,0 +1,148 @@
+| 철인3종   | 철인3종 완주 기반 자동 부여                       |
+| 사이클    | 사이클 대회 완주 기반 자동 부여                   |
+| 제네럴    | 가입 기간·계절 참여·복합 활동 기반 자동 부여      |
+| 수여      | 운영진이 직접 부여 (자유 생성 가능)              |
+| 이벤트    | 특정 이벤트·시즌 참여로 해금, 전체 카운트 미포함 |
+| 마일리지런 | 마일리지런 이벤트 참여·활동 기록 기반 자동 부여  |
+
+---
+
+## 칭호 목록
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)